scripts/testing/setup-mock-test-env.py

```diff
@@ -155,6 +155,13 @@ def parse_args():
     group.add_argument('--install', '-i', metavar='<packages>', action='store', type=str,
                        dest='install',
                        help="""install additional packages to the mock""")
+    group.add_argument('--install-pip', '-P', metavar='<pip packages>', action='store', type=str,
+                       dest='install_pip', default=None,
+                       help="""
+                       install pip additional packages in the mock
```

Comment:
Just an enhancement suggestion:

"install additional packages from  Python Package Index repository to the mock environment via the pip tool"

Reply:
Good idea, thanks!